<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Снятие показаний</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="5">
         <widget class="QLineEdit" name="lineEdit_Patronymic_2"/>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_Name_2"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_SurName_2"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButton_Enter">
          <property name="text">
           <string>Вход</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="7">
         <widget class="QFrame" name="frame">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="8" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Diastol">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_BeginMetering">
             <property name="text">
              <string>Начать измерение</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Pulse">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_Date">
             <property name="text">
              <string>Дата</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_Systol">
             <property name="text">
              <string>Систолическое давление</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_Temperature">
             <property name="text">
              <string>Температура</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_Pulse">
             <property name="text">
              <string>Пульс</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Systol">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Temperature">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Date">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_Diastol">
             <property name="text">
              <string>Диастолическое давление</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_Time">
             <property name="text">
              <string>Время</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Time">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QPushButton" name="pushButton_AddRecord">
             <property name="text">
              <string>Добавить запись</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Просмотр данных</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="5">
         <widget class="QLineEdit" name="lineEdit_Patronymic"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Surname">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_SurName"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_Name">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButton_Show">
          <property name="text">
           <string>Отобразить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_Name"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_Patronymic">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="7">
         <widget class="QTableWidget" name="tableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Время</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Температура</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Пульс</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cист давление</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Диас давление</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_SaveDB"/>
    <addaction name="action_LoadDB"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_Qt"/>
    <addaction name="action_AboutProgram"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_Qt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="action_AboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_SaveDB">
   <property name="text">
    <string>Сохранить БД</string>
   </property>
  </action>
  <action name="action_LoadDB">
   <property name="text">
    <string>Загрузить БД</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
